cluster/gce/gci/configure-helper.sh

```diff
@@ -163,12 +163,12 @@ function setup-logrotate() {
   # * keep only 5 old (rotated) logs, and will discard older logs.
   cat > /etc/logrotate.d/allvarlogs <<EOF
 /var/log/*.log {
-    rotate 5
+    rotate ${LOGROTATE_FILES_NO:-5}
```

Comment:
Maybe it's better to set this default inside config-default.sh rather than hiding it here.